src/main/java/com/SAS/soccer_association_system/report/GameReport.java

```diff
@@ -0,0 +1,61 @@
+package com.SAS.soccer_association_system.report;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+/**
+ * The class represents the game report that made by the referee when a game ends
+ */
+public class GameReport {
+
+    private LocalDate gameDate;
+    private LocalDateTime time;
```

Comment:
I think the name of the variable should be updatedTime - as for the time the report updated for the last time.

Reply:
And also add a variable of description of the report.

Reply:
I added description.
time param removed and the game report time will be as the game-ending time.